common/presenters/move-to-meta-list-component.test.js

```diff
@@ -160,93 +173,6 @@ describe('Presenters', function () {
       })
     })
 
-    context('with only `date from`', function () {
```

Comment:
I think we need to keep this behaviour for moves which don't have a date, in that case we don't even need to show the journeys summary as the `move.date` will be `null`.